app/playground/page.tsx

```diff
@@ -7,36 +7,28 @@ export const metadata: Metadata = {
   icons: 'EDPN_logo_black.png',
```

Comment:
```suggestion
  icons: 'EDPN_logo_dark_background.png',
```
I think I may have told you to use logo_black which is wrong. logo_black isn't used yet. We may never use it, or we may like it for light-theme use in the future.